libsolc/libsolc.cpp

```diff
@@ -270,48 +84,18 @@ static string s_outputBuffer;
 
 extern "C"
 {
-extern char const* license() noexcept
+extern char const* solidity_license() noexcept
 {
 	static string fullLicenseText = otherLicenses + licenseText;
 	return fullLicenseText.c_str();
 }
-extern char const* version() noexcept
-{
-	return VersionString.c_str();
-}
-extern char const* compileJSON(char const* _input, bool _optimize) noexcept
-{
-	s_outputBuffer = compileSingle(_input, _optimize);
-	return s_outputBuffer.c_str();
-}
-extern char const* compileJSONMulti(char const* _input, bool _optimize) noexcept
-{
-	s_outputBuffer = compileMulti(_input, _optimize);
-	return s_outputBuffer.c_str();
-}
-extern char const* compileJSONCallback(char const* _input, bool _optimize, CStyleReadFileCallback _readCallback) noexcept
-{
-	s_outputBuffer = compileMulti(_input, _optimize, _readCallback);
-	return s_outputBuffer.c_str();
-}
-extern char const* compileStandard(char const* _input, CStyleReadFileCallback _readCallback) noexcept
```

Comment:
I'm kind of leaning towards keeping this single line for now (during the release candidate time) so that projects can easily test it with truffle.